src/effects.md

```diff
@@ -104,7 +104,7 @@ function `h` that accepts two function arguments `f`
 and `g` of which at most one is impure:
 
 ```flix
-def h(f: a -> b & ef1, g: b -> c & (not ef1 or ef2)): Unit
+def h(f: a -> b \ ef1, g: b -> c \ { (not ef1) or ef2 }): Unit
```

Comment:
I need some help to make this correct:
```
def h(f: a -> b & ef1, g: b -> c & (not ef1 or ef2)): Unit
```

Reply:
Ah, this example cannot be updated, so just leave it.